### PR TITLE
added support for specifying class labels different than +/- 1

### PR DESCRIPTION
**Pull request by andreh7**
which is useful e.g. when using a binary cross entropy loss function

### CODE REVIEW

**Comment by hpenedones** [comment]
Thanks, looks good to me!
